Parsers/WindowsEvent/README.md

```diff
@@ -0,0 +1,22 @@
+# Microsoft Windows Events ASIM parsers
+
+This template deploys all [Microsoft Windows Event] ASIM parsers. The template is part of the [Azure Sentinel Information Mode (ASIM)](https://aka.ms/AzSentinelNormalization).The Azure Sentinel Information Mode (ASIM) enables you to use and create source-agnostic content, simplifying your analysis of the data in your Azure Sentinel workspace.
+
+For more information, see:
+
+- [Azure Defender for IoT](https://docs.microsoft.com/en-us/windows/security/threat-protection/auditing/advanced-security-auditing-faq/) 
+- [Normalization and the Azure Sentinel Information Model (ASIM)](https://aka.ms/AzSentinelNormalization)
+
+<br>
+
+[![Deploy to Azure](https://aka.ms/deploytoazurebutton)](https://aka.ms/AzSentinelWindowsEventARM)
+
+<br>
+
+The template deploys the following:
```

Comment:
I'd rather do the more textual version we moved to in recent updated.

Reply:
any example?